terraspace/app/stacks/bucket-to-indexer-lambda/main.tf

```diff
@@ -0,0 +1,43 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.38"
+    }
+  }
+
+  required_version = ">= 1.0.0"
+}
+
+data "terraform_remote_state" "indexing" {
+  backend = "s3"
+  config = {
+    bucket = "<%= expansion('ipfs-ep-terraform-state-:ACCOUNT-${var.indexing_stack_region}') %>"
+    key    = "<%= expansion('${var.indexing_stack_region}/:ENV/stacks/indexing/terraform.tfstate') %>"
```

Comment:
I wonder if we should expose `account_id` and `env` as standard terraform `local` references, so we don't have to mix terraspace erb with terraform interpolations.  My thought process here is... "Try to avoid using ERB within our stack's `*.tf` files."

These lines would end up becoming:
```hcl
bucket = "ipfs-ep-terraform-state-${local.account_id}-${var.indexing_stack_region}"
key    = "${var.indexing_stack_region}/${local.env}/stacks/indexing/terraform.tfstate"
```